backend/metasfresh-webui-api/src/main/java/de/metas/ui/web/window/descriptor/sql/ProductLookupDescriptor.java

```diff
@@ -357,6 +356,11 @@ private static StringBuilder appendFilterByIsActive(final StringBuilder sqlWhere
 		return sqlWhereClause.append("\n p.").append(I_M_Product_Lookup_V.COLUMNNAME_IsActive).append("=").append(sqlWhereClauseParams.placeholder(true));
 	}
 
+	private static StringBuilder appendFilterByDiscontinued(final StringBuilder sqlWhereClause, final SqlParamsCollector sqlWhereClauseParams)
```

Comment:
From looking at the code, i'm not sure that we can still enter discontinued products at not-order-places.
From a quick check, those include invoice lines and forecast lines; maybe more.

Please make sure that this still works

Reply:
Modified ProductLookupDescriptor so to only show discontinued products when the new boolean param is set to false.